editors/vscode/client/extension.ts

```diff
@@ -242,7 +264,17 @@ export async function activate(context: ExtensionContext) {
     myStatusBarItem.backgroundColor = bgColor;
   }
   updateStatsBar(configService.config.enable);
-  client.start();
+  await client.start();
+
+  const initialConfigFiles =
+    (await Promise.all([oxlintConfigFileGlob, ...workspaceConfigPatterns].map(async globPattern => {
+      return workspace.findFiles(globPattern);
+    }))).flat();
+  await client.sendNotification(DidChangeWatchedFilesNotification.type, {
+    changes: initialConfigFiles.map(file => {
+      return { uri: file.toString(), type: FileChangeType.Created };
+    }),
+  });
```

Comment:
On startup, send all found config files to the language server as "created", so that they get added to the nested config map.